--- fys_lib.f.orig	2005-06-14 05:33:24.000000000 -0700
+++ fys_lib.f	2022-10-16 15:39:57.308759169 -0700
@@ -73,6 +73,8 @@
 c
       data nmass /257/
 c
+      call getenv("GRAZ_MASS_EXP",mass_exp)
+      call getenv("GRAZ_MASS_NIX",mass_nix)
       if(izd.lt.0 .or. iad.lt.0 .or. iad.lt.izd)then
        write(*,*)' Error: wrong input in masse_s'
        ierr = 1
@@ -184,6 +186,7 @@
 c
       include 'grazing_file.icl'
 c
+      call getenv("GRAZ_FILE_BE23",file_be23)
       open (unit=nodat,file=file_be23,status='old')
       istat = ierr
       iz = ize
